src/plugins/vis_type_vislib/kibana.json

```diff
@@ -3,6 +3,6 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["charts", "data", "expressions", "visualizations", "kibanaLegacy"],
+  "requiredPlugins": ["charts", "data", "expressions", "visualizations", "kibanaLegacy", "indexPatternManagement"],
```

Comment:
not sure that `vis_type_vislib` should depends on `indexPatternManagement`

Reply:
You are right. Removed it.